beta/src/components/MDX/Challenges/Navigation.tsx

```diff
@@ -16,14 +16,14 @@ export function Navigation({
 }: {
   challenges: ChallengeContents[];
   handleChange: (id: string) => void;
-  currentChallenge?: ChallengeContents;
+  currentChallenge: ChallengeContents;
   isRecipes?: boolean;
 }) {
   const containerRef = React.useRef<HTMLDivElement>(null);
   const challengesNavRef = React.useRef(
     challenges.map(() => createRef<HTMLButtonElement>())
   );
-  const scrollPos = (currentChallenge?.order || 1) - 1;
+  const scrollPos = (currentChallenge.order || 1) - 1;
```

Comment:
you can drop `|| 1` now?

Reply:
Apologies. That's what happens when I try and fit a refactor into a 15min work break... Fixed now.